packages/components/src/components/Log/Log.js

```diff
@@ -25,21 +25,44 @@ const LogLine = ({ data, index, style }) => (
   </div>
 );
 
-class Log extends Component {
-  getLogList() {
-    const { status, intl } = this.props;
+const trailers = {
+  Completed: {
+    id: 'dashboard.pipelineRun.stepCompleted',
+    defaultMessage: 'Step completed'
+  },
+  Error: {
+    id: 'dashboard.pipelineRun.stepFailed',
+    defaultMessage: 'Step failed'
+  }
+};
+
+export class LogContainer extends Component {
+  state = { loading: true };
+
+  componentDidMount() {
+    this.loadLog().then(this.setState({ loading: false }));
```

Comment:
Let's move the state update into `loadLog` where it's adding the logs to state, so this would just become:

```
this.loadLog();
this.initPolling();
```

and in `loadLog`:

`this.setState({ loading: false, logs: logs ? logs.split('\n') : undefined });`

This makes sure we're covered if we add a manual refresh or similar in future.